controller-norbac.jsonnet

```diff
@@ -37,6 +37,7 @@ local namespace = 'kube-system';
                 readOnlyRootFilesystem: true,
                 runAsNonRoot: true,
                 runAsUser: 1001,
+                fsGroup: 65534,
```

Comment:
From my testing, this field is not allowed in the securityContext of a containerSpec.  It needs to be in a new securityContext section of the podSpec template.

Reply:
indeed

```
To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
error: error validating "controller.yaml": error validating data: ValidationError(Deployment.spec.template.spec.containers[0].securityContext): unknown field "fsGroup" in io.k8s.api.core.v1.SecurityContext; if you choose to ignore these errors, turn validation off with --validate=false
```

sorry I was lazy and wanted the testing system to guide me.
Fixed and run through `kubecfg validate` to pre-check.

PTAL